src/seml/manage.py

```diff
@@ -704,6 +744,7 @@ def print_status(
             str(record['count']),
             ", ".join(map(slice_to_str, to_slices(record['ids']))),
             ", ".join(map(slice_to_str, to_slices(record['batch_ids']))),
+            *([str(len(set(record['ids']) & duplicate_experiment_ids))] if detect_duplicates_ else []),
```

Comment:
This right now prints both experiments as duplicates, wouldn't it semantically make sense that one experiment is always the base and the others are the duplicates?

Reply:
Which one would be the base though? The one with the smaller `_id`? I see your point, though, if we want to define this as the base we can change the output accordingly.